<?xml version="1.0"?>
<launch>
    <!-- Right arm: -->
    <include 
    file="$(find kinova_positional_control)/launch/oculus_mapping.launch">
        <arg 
        name="robot_name" 
        value="right_arm"/>
    </include>

    <!-- Right arm: -->
    <include 
    file="$(find kinova_positional_control)/launch/oculus_mapping.launch">
        <arg 
        name="robot_name" 
        value="left_arm"/>
    </include>
</launch>